packages/block-library/src/navigation/block.json

```diff
@@ -45,10 +53,14 @@
 		"customFontSize": "customFontSize",
 		"showSubmenuIcon": "showSubmenuIcon",
 		"style": "style",
-		"orientation": "orientation"
+		"orientation": "orientation",
+		"isResponsive": "isResponsive"
```

Comment:
Why do we need this here? I couldn't find anywhere this is consumed by `context`.

Reply:
It's actually leftovers, thanks for spotting it.